openedxstats/apps/sites/templates/sites/navbar.html

```diff
@@ -0,0 +1,13 @@
+<nav class="navbar navbar-inverse navbar-fixed-top">
```

Comment:
It never hurts to add a comment to the top of a file explaining its place in the universe.